discord/commands/options.py

```diff
@@ -138,6 +148,10 @@ class Option:
         .. note::
 
             Does not validate the input value against the autocomplete results.
+    channel_types: Optional[List[:class:`discord.ChannelType`]]
```

Comment:
We haven't manually gone in and updated old docstrings to the new annotation syntax, but for new entries we should definitely use it.
```suggestion
    channel_types: list[:class:`discord.ChannelType`] | None
```

Reply:
agreed